images/openshift-installer/README.md

```diff
@@ -1,4 +1,4 @@
-# tectonic-installer
+# openshift-installer
 
 [![Container Repository on Quay](https://quay.io/repository/coreos/tectonic-installer/status "Container Repository on Quay")](https://quay.io/repository/coreos/tectonic-installer)
```

Comment:
We should probably update this too.

Reply:
I'm going to leave that to future PR cuz we need to figure out the quay repo and credential, etc.